src/maven_artifact/requestor.py

```diff
@@ -1,23 +1,31 @@
 import base64
 import requests
 
+from maven_artifact.utils import Utils
+
 
 class RequestException(Exception):
     def __init__(self, msg):
         self.msg = msg
 
 
 class Requestor(object):
-    def __init__(self, username=None, password=None, user_agent="Maven Artifact Downloader/1.0"):
+    def __init__(self, username=None, password=None, token=None, user_agent="Maven Artifact Downloader/1.0"):
```

Comment:
we should probably get the version string here

Reply:
Yes. I spend a serious amount on time on this and decided to just wait (perhaps create a separate ticket). You can use the metadata library to get the version. It will work when the tool is ran from an installed module but not otherwise.

I'll provide links later tonight possibly.

Reply:
Link added to #25.